packages/cli/__tests__/zosuss/__unit__/issue/ssh/Ssh.handler.unit.test.ts

```diff
@@ -9,12 +9,13 @@
 *
 */
 
-jest.mock("@zowe/zos-uss-for-zowe-sdk");
+jest.mock("../../../../../../zosuss/lib/Shell");
+
 import { IHandlerParameters, IProfile, CommandProfiles } from "@zowe/imperative";
 import * as SshHandler from "../../../../../src/zosuss/issue/ssh/Ssh.handler";
 import * as SshDefinition from "../../../../../src/zosuss/issue/ssh/Ssh.definition";
 import { getMockedResponse } from "../../../../../../../__tests__/__src__/mocks/ZosmfProfileMock";
-import { Shell } from "@zowe/zos-uss-for-zowe-sdk";
+import * as Shell from "@zowe/zos-uss-for-zowe-sdk";
```

Comment:
I'm assuming this was done in order to `jest.mock` it?

Reply:
I don't know why I was doing that.